bot.py

```diff
@@ -29,19 +31,40 @@
 intents = discord.Intents(guilds=True, members=True, bans=True, messages=True, typing=True)
 activityStatus = discord.Activity(type=discord.ActivityType.playing, name='DM to contact mods')
 bot = commands.Bot(
-    config.command_prefixes, fetch_offline_members=True, activity=activityStatus, case_insensitive=True, intents=intents
+    command_prefix=commands.when_mentioned,
+    fetch_offline_members=True,
+    activity=activityStatus,
+    case_insensitive=True,
+    intents=intents,
 )
+slash = SlashCommand(bot)
+guildList = [config.guild]
 
 
 class Mail(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
         self.READY = False
         self.closeQueue = {}
-
-    @commands.has_any_role(config.modRole)
-    @commands.command(name='close')
-    async def _close(self, ctx, delay: typing.Optional[str]):
+        loop = bot.loop
+        loop.create_task(slash.sync_all_commands(delete_from_unused_guilds=True))
+
+    @cog_ext.cog_slash(
+        name='close',
+        guild_ids=guildList,
+        description='Closes a modmail thread, optionally with a delay',
+        permissions={config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]},
```

Comment:
`{config.guild: [create_permission(config.modRole, SlashCommandPermissionType.ROLE, True)]}` should be made into a static variable and passed instead.